src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+  }
+  static alive = [];
```

Comment:
The `alive` static property is shared across all instances of `Animal`, `Herbivore`, and `Carnivore`. This means that all animals, regardless of type, will be stored in the same array. If this is not the intended behavior, consider using instance-specific properties or separate arrays for different types of animals.